docs/guide/distancesensor-sensors.md

```diff
@@ -0,0 +1,146 @@
+
+### Sharp
+
+#### Sharp GP2D120
+
+%figure "Sharp GP2D120 model"
+
+![sharp_GP2D120.png](images/sensors/sharp_GP2D120.png)
+
+%end
+
+The [Sharp GP2D120](https://www.pololu.com/file/0J157/GP2D120-DATA-SHEET.pdf) is a distance measuring sensor with integrated signal processing and analog voltage output.
+This device outputs the voltage corresponding to the detection distance.
+
+The model has the following specifications:
+
+- `effective range`: 4 to 30 [cm]
+- `output type`: analog
+- `response time`: 39 [ms]
+- `start up delay`: 44 [ms]
+- `average consumption`: 33 [mA]
+
+```
+PROTO SharpGP2D120 {
+  SFVec3f     translation  0 0 0
+  SFRotation  rotation     0 1 0 0
+  SFString    name         "Sharp's IR sensor GP2D120"
+  SFString    model        "GP2D120"
+}
+```
+
+The `lookupTable` field of the [DistanceSensor](../reference/distancesensor.md) is already implemented according to it characteristics found in his [datasheet](https://www.pololu.com/file/0J157/GP2D120-DATA-SHEET.pdf).
+
+The function `wb_distance_sensor_get_value` return the voltage/intensity of the measurement. To convert these values, use the following formulas:
+- Given meter, the result expected is in voltage: `y(x) = 0.5131*x^(-0.5735)-0.6143`
+- Given voltage, the result expected is in meter: `y(x) = 0.1594*x^(-0.8533)-0.02916`
+
+#### Sharp GP2Y0A02YK0F
+
+%figure "Sharp GP2Y0A02YK0F model"
+
+![sharp_GP2Y0A02YK0F.png](images/sensors/sharp_GP2Y0A02YK0F.png)
+
+%end
+
+The [Sharp GP2Y0A02YK0F](https://global.sharp/products/device/lineup/data/pdf/datasheet/gp2y0a02yk_e.pdf) is a distance measuring sensor unit, composed of an integrated combination of PSD (position sensitive detector), IR-LED (infrared emitting diode) and signal processing circuit.
+The variety of the reflectivity of the object, the environmental temperature and the operating duration are not influenced easily to the distance detection because of adopting the triangulation method.
+This device outputs the voltage corresponding to the detection distance.
+So this sensor can also be used as a proximity sensor.
+
+The model has the following specifications:
+
+- `effective range`: 20 to 150 [cm]
+- `output type`: analog
+- `response time`: 33 [ms]
+- `start up delay`: 50 [ms]
+- `average consumption`: 33 [mA]
+
+```
+PROTO SharpGP2Y0A02YK0F {
+  SFVec3f     translation  0 0 0
+  SFRotation  rotation     0 1 0 0
+  SFString    name         "Sharp's IR sensor GP2Y0A02YK0F"
+}
+```
+
+The `lookupTable` field of the [DistanceSensor](../reference/distancesensor.md) is already implemented according to it characteristics found in his [datasheet](https://global.sharp/products/device/lineup/data/pdf/datasheet/gp2y0a02yk_e.pdf).
+
+The function `wb_distance_sensor_get_value` return the voltage/intensity of the measurement. To convert these values, use the following formulas:
+- Given meter, the result expected is in voltage: `y(x) = 1.784*x^(-0.4215)-1.11`
```

Comment:
As in the previous comment I would simplify this with 
```suggestion
- Convert meters to voltage: `y(x) = 1.784*x^(-0.4215)-1.11`
```

same for the next line